src/custom/constants/index.ts

```diff
@@ -63,6 +63,12 @@ export const V_COW_CONTRACT_ADDRESS: Record<number, string> = {
   [ChainId.RINKEBY]: '0x5Bf4d1f8d1cB35E0aeA69B220beb97b8807504eA',
 }
 
+export const COW_CONTRACT_ADDRESS: Record<number, string> = {
+  [ChainId.MAINNET]: '0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB',
+  [ChainId.XDAI]: '0x177127622c4A00F3d409B75571e12cB3c8973d3c',
+  [ChainId.RINKEBY]: '0x5Bf4d1f8d1cB35E0aeA69B220beb97b8807504eA', // same as V_COW for now, SHOULD BE UPDATED!!!
```

Comment:
nit: comment using the same standard: `// TODO: blah blah` so anyone else later can find by keyword

Reply:
second these 👌

Reply:
Good point to use TODO in such cases. I've updated the address here